fastlane/lib/fastlane/actions/appetize_url_generator.rb

```diff
@@ -0,0 +1,107 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class AppetizeUrlGeneratorAction < Action
```

Comment:
What about `AppetizeViewingUrlGenerator` to clarify what this is different than the `Appetize` action.